src/useAntdTable/index.ts

```diff
@@ -289,15 +338,26 @@ export default function useAntdTable<Result, Item>(
     });
   }, [state.searchType]);
 
-  // 表格翻页
+  // 表格翻页 排序 筛选等
   const changeTable = useCallback(
-    (e: PaginationConfig) => {
+    (
+      p: PaginationConfig,
+      f: Record<keyof Item, string[]> = {} as Record<keyof Item, string[]>,
+      s: SorterResult<Item> = {} as SorterResult<Item>,
+    ) => {
+      /* 如果 filter，或者 sort 变化，就初始化 current */
+      const needReload =
+        JSON.stringify(f) !== JSON.stringify(state.filters) ||
```

Comment:
we can use [react-fast-compare](https://www.npmjs.com/package/react-fast-compare) to compare objects that may have different key order

Reply:
fixed use lodash.isEqual